config/v2/config.go

```diff
@@ -56,6 +56,7 @@ type Common struct {
 	Owner            *string           `yaml:"owner,omitempty"`
 	Project          *string           `yaml:"project,omitempty"`
 	Providers        *Providers        `yaml:"providers,omitempty"`
+	RemoteStates     []string          `yaml:"remote_states,omitempty"`
```

Comment:
I think we want the ability to limit account remote state as well as remote states. I was thinking something like:

```
depends_on:
  accounts:
  - foo
  - bar
  components:
  - baz
```

Reply:
My implementation will do that (because we use the same list to filter both, though the interface is probably not clear. I like your interface better.